### DISCUSSION TITLE
Advice

### DISCUSSION THREAD

**Discussion by Usama-Ismail** [OrchardCMS/OrchardCore#16260] [Q&A] [answered]
How to get started with Orchard Core like what should you do first understand the application by using it or dive straightly into development?

**Comment by Piedone** [comment, marked as answer]
I suggest you first check out https://docs.orchardcore.net/en/latest/, then https://docs.orchardcore.net/en/latest/getting-started/ and https://docs.orchardcore.net/en/latest/resources/tutorials/.

**Reply by Usama-Ismail** [reply]
thanks @Piedone
